spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    first_name { "MyString" }
+    last_name { "MyString" }
+    username { "MyString" }
+    password_hash { "MyString" }
+    phone_number { "MyString" }
+    email { "MyString" }
```

Comment:
as above, we'd normally set some useful values here, e.g.
```ruby
first_name { Faker::Name.first_name }
last_name { Faker::Name.last_name }
email { Faker::Internet.email }
```

Reply:
That's really interesting to see! I'll need to have another chat with you in regards to the factories file for RSpec.